sefaria/model/lexicon.py

```diff
@@ -141,8 +146,14 @@ def get_sense(self, sense):
         return text
 
     def headword_string(self):
-        return ', '.join(
-            ['<strong dir="rtl">{}</strong>'.format(hw) for hw in [self.headword] + getattr(self, 'alt_headwords', [])])
+        headwords = [self.headword] + getattr(self, 'alt_headwords', [])
+        if self.contents()['parent_lexicon_details']['version_lang'] == 'he':
```

Comment:
There seems to be a large inefficiency in `contents()` it loads the `Lexicon` from the database, but it's accessible by `self.parent.lexicon`. Use `self.parent.lexicon` instead of `contents()['parent_lexicon_details']` and can you fix `contents()` so that it uses `self.parent.lexicon`?